src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -210,9 +217,23 @@ export class StoreQuickstartSetup extends SfdxCommand {
             )
             .toString()
             .replace('YourCommunitySiteNameHere', this.varargs['communitySiteName'] as string)
-            .replace('YourCommunityExperienceBundleNameHere', this.varargs['communityExperienceBundleName'] as string)
             .replace('YourCommunityNetworkNameHere', this.varargs['communityNetworkName'] as string)
             .replace('ApiVersionHere', this.flags.apiversion as string);
+
+        if (!this.isUsingDigitalExperienceBundle) {
+            packageRetrieve = packageRetrieve.replace(
+                'YourCommunityExperienceBundleNameHere',
+                this.varargs['communityExperienceBundleName'] as string
+            );
+        } else {
+            // Digital Experience Bundle needs 'site/' prepended to the community name.
+            packageRetrieve = packageRetrieve
+                .replace(
+                    'YourCommunityExperienceBundleNameHere',
+                    `site/${this.varargs['communityExperienceBundleName'] as string}`
+                )
+                .replace('ExperienceBundle', 'DigitalExperienceBundle');
```

Comment:
Why do we need to replace 'ExperienceBundle' with 'DigitalExperienceBundle'. I thought it should by default has 'DigitalExperienceBundle'

Reply:
It gets the retrieve template from quickstart config folder which has ExperienceBundle on it (which will break the metadata retrieval). Instead of having new templates, I am just editing the template in runtime.